www/_data/core.yml

```diff
@@ -1,21 +1,18 @@
 -
-  name: Shrine (ClassMethods)
-  path: Plugins/Base/ClassMethods.html
--
-  name: Shrine (InstanceMethods)
-  path: Plugins/Base/InstanceMethods.html
+  name: Shrine
+  path: Shrine.html
 -
   name: Shrine::UploadedFile
-  path: Plugins/Base/FileMethods.html
+  path: Shrine/UploadedFile.html
 -
   name: Shrine::Attacher
-  path: Plugins/Base/AttacherMethods.html
+  path: Shrine/Attacher.html
 -
   name: Shrine::Attachment
-  path: Plugins/Base/AttachmentMethods.html
+  path: Shrine/Attachment.html
```

Comment:
I would prefer if we referenced the `InstanceMethods.html` for these three core classes, because they will hold the meat of the implementation (the `ClassMethods` module will only hold a few methods which I don't think are important to look at).